Model/Exceptions/NotFoundException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Model.Exceptions
+{
+    public class NotFoundException : BaseException
+    {
+        public NotFoundException(string errorMassage) : base(errorMassage)
+        {
+            base.StatusCode = 404;
```

Comment:
It would be cleaner if you created `enum` for each status code, e.g. `NotFound = 404` and then use `NotFound` in this class